site/data/s3.ts

```diff
@@ -31,7 +31,7 @@ export const putItem = async (
     }
 };
 
-export const getItem = async (bucket: string, key: string, originalFilename: string): Promise<string> => {
+export const getS3SignedUrl = async (bucket: string, key: string, originalFilename: string): Promise<string> => {
     logger.info("", {
         context: "data.s3",
         message: "getting item from s3",
```

Comment:
Should we change the logger message and error message to "getting signed url from s3" - just thinking that `getS3SignedUrl` and `getObject` has the same error message which may make debugging annoying?